app/derivative_services/scanned_map_derivative_service.rb

```diff
@@ -42,6 +42,9 @@ def parent
   def create_derivatives
     jp2_derivative_service.create_derivatives if jp2_derivative_service.valid?
     thumbnail_derivative_service.create_derivatives if thumbnail_derivative_service.valid?
+  rescue MiniMagick::Error => mini_magick_error
+    Rails.logger.error "Failed to generate JP2000 derivatives for #{id}: #{mini_magick_error.message}"
```

Comment:
This will prevent the job from restarting, which means if it fails it will always be broken until we manually kick it. Is that what we want?

Reply:
Sorry, should the errors *not* be rescued then, and this should propagate the error?  I can still clean up the derivatives and ensure that the error message is logged.

Reply:
Maybe we should just do this for all derivative errors? If a StaleObjectError happens we could end up with a duplicate too.